File: BeautySaloon/BeautySaloon.API/Entities/BeautySaloonContextEntities/Reservation.cs
﻿namespace BeautySaloon.API.Entities.BeautySaloonContextEntities;

public class Reservation
{
    public int Id { get; set; }
    public int ServiceId { get; set; }
    public Service? Service { get; set; }
    public DateTime DateTime { get; set; }
    public string CustomerPhoneNumber { get; set; }
    public int MasterId { get; set; }
    public Master? Master { get; set; }
}

File: BeautySaloon/BeautySaloon.Angular/src/app/shared/models/resrvation/reservation.ts
import { Customer } from "../customer";
import { Master } from "../master";
import { Service } from "../service";

export interface Reservation {
  id: number;
  serviceId: number;
  service: Service | null;
  dateTime: Date;
  customerPhoneNumber: string;
  customer: Customer | null;
  masterId: number;
  master: Master | null;
}